Source/objects.cpp

```diff
@@ -1288,27 +1288,33 @@ void AddBarrel(Object &barrel)
 void AddShrine(Object &shrine)
 {
 	shrine._oRndSeed = AdvanceRndSeed();
-	bool slist[NumberOfShrineTypes];
-
 	shrine._oPreFlag = true;
 
-	int shrines = gbIsHellfire ? NumberOfShrineTypes : 26;
+	int shrineCount = gbIsHellfire ? NumberOfShrineTypes : 26;
+	bool slist[NumberOfShrineTypes] = { false };

```

Comment:
```suggestion
	bool slist[NumberOfShrineTypes] = { };
```
{false} only sets first element to false